packages/mask/src/plugins/FindTruman/SNSAdaptor/OptionsCard.tsx

```diff
@@ -251,25 +260,31 @@ export default function OptionsCard(props: OptionsViewProps) {
     const renderSubmitButton = (userStatus: UserPuzzleStatus | UserPollStatus) => {
         return (
             <div style={{ textAlign: 'right', marginTop: '8px', paddingBottom: '8px' }}>
-                <LoadingButton
-                    disabled={userStatus.status === 0 || choice === -1}
-                    onClick={() => {
-                        setSubmitting(true)
-                        onSubmit(choice)
-                            .then((res) => {
-                                setSubmitting(false)
-                            })
-                            .catch((error) => {
-                                setSnackVisible(true)
-                                setSubmitting(false)
-                            })
-                    }}
-                    endIcon={<Send />}
-                    loading={submitting}
-                    loadingPosition="end"
-                    variant="contained">
-                    {t(userStatus.status === 0 ? 'plugin_find_truman_vote_finish' : 'plugin_find_truman_submit')}
-                </LoadingButton>
+                {selected ? (
+                    <Button variant="contained" disabled={true} color="success" endIcon={<DoneOutlined />}>
+                        {t('plugin_find_truman_submitted')}
+                    </Button>
+                ) : (
+                    <LoadingButton
```

Comment:
Refactor to use `<ActionButtonPromise />`.

Reply:
Resolved.